target/linux/ath79/nand/base-files/etc/board.d/01_leds

```diff
@@ -11,6 +11,10 @@ glinet,gl-ar300m-nand|\
 glinet,gl-ar300m-nor)
 	ucidef_set_led_netdev "lan" "LAN" "green:lan" "eth0"
 	;;
+kt,dw02-412h-64m|\
+kt,dw02-412h-128m)
+	ucidef_set_led_netdev "wan" "WAN" "green:lan" "eth0.2"
```

Comment:
?

Reply:
Okay. Then... should I use this trigger? :
`ucidef_set_led_switch`

Reply:
yes.